docs/source/configuration.rst

```diff
@@ -253,15 +253,27 @@ Flood Prevention
 ----------------
 
 In order to avoid flooding the server, Sopel has a built-in flood prevention
-mechanism. It can be controlled with several directives:
+mechanism. The flood burst limit can be controlled with these directives:
 
 * :attr:`~CoreSection.flood_burst_lines`: the number of messages
   that can be sent before triggering the throttle mechanism.
-* :attr:`~CoreSection.flood_empty_wait`: time to wait once burst limit has been
-  reached before sending a new message.
 * :attr:`~CoreSection.flood_refill_rate`: how much time (in seconds) must be
   spent before recovering flood limit.
 
+The wait time when the flood limit is reached can be controlled with these:
+
+* :attr:`~CoreSection.flood_empty_wait`: time to wait once burst limit has been
+  reached before sending a new message.
+* :attr:`~CoreSection.flood_max_wait`: maximum time to wait once burst limit
+  has been reached before sending a new message. This limit includes the extra
+  penalty for longer messages.
```

Comment:
```suggestion
* :attr:`~CoreSection.flood_max_wait`: absolute maximum time to wait before
  sending a new message once the burst limit has been reached.
```

I just think it's clearer to say this is an "absolute maximum", rather than referencing a concept not yet described ("extra penalty for longer messages").

Reply:
Tbh I was kind of annoyed that I had to talk about extra penalty here before it was defined. So it's a good thing.